HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -248,6 +248,11 @@ def customizeHLTfor40264(process):
 
     return process
 
+def customizeHLTfor40443(process):
+     for producer in esproducers_by_type(process, "TrackerAdditionalParametersPerDetESModule"):
+	delattr(producer, "TrackerAdditionalParametersPerDetESModule")	
```

Comment:
Please use the following or equivalent (and do not use tabs.. [*]).
```suggestion
     for foo in [bar for bar in esproducers_by_type(process, "TrackerAdditionalParametersPerDetESModule")]:
         delattr(process, foo.label())
```
[*]
```
Traceback (most recent call last):
[..]
    delattr(producer, "TrackerAdditionalParametersPerDetESModule")
TabError: inconsistent use of tabs and spaces in indentation
```